roles/lib_jobs/tasks/main.yml

```diff
@@ -1,44 +1,45 @@
 ---
 # tasks file for roles/lib_jobs
 # needed for capistrano to have access
-- name: Install site configuration
-  template:
+- name: lib_jobs | Install site configuration
+  ansible.builtin.template:
     src: 'ruby_app_config'
     dest: '/home/{{ deploy_user }}/app_configs/{{ ruby_app_name }}'
     owner: '{{ deploy_user }}'
     group: '{{ deploy_user }}'
     mode: 0644
 
-- name: Create app directory structure
-  file:
+- name: lib_jobs | Create app directory structure
+  ansible.builtin.file:
     path: '/opt/{{ app_directory }}/shared/tmp'
     state: 'directory'
     owner: '{{ deploy_user }}'
     group: '{{ deploy_user }}'
     mode: 0755
 
-- name: Install dependencies
```

Comment:
Double checked that we no longer need this for the SFX code base.  It was utilized for Voyager and can be removed.